reference/docs-conceptual/community/contributing/editing-cmdlet-ref.md

```diff
@@ -16,9 +16,9 @@ errors that must be fixed before we can accept your contribution.
 
 ## General guidelines
 
-- Do not remove any of the header structures. PlatyPS expects a specific set of headers.
-- The **Input type** and **Output type** headers must have a type. If the cmdlet does not take input
-  or return a value then use the value `None`.
+- Don't remove any of the ATX header structures. PlatyPS expects a specific set of headers.
```

Comment:
Shoulld the ATX acronym have a definition in parenthesis for clarification?

Reply:
updated